.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateTimeExtractor.cs

```diff
@@ -307,19 +306,6 @@ public List<Token> SpecialTimeOfDate(string text, DateObject reference)
             return ret;
         }
 
-        // Special case for 'the end of today'
-        public List<Token> SpecialTimeOfDay(string text, DateObject reference)
-        {
-            var ret = new List<Token>();
-            var match = this.config.SpecificEndOfRegex.Match(text);
```

Comment:
I guess this regex is now references somewhere else?

Reply:
yes, it is used in the method SpecialTimeOfDate which correctly extracts pattern like "the end of today".